Doppler.MercadoPagoApi/appsettings.int.json

```diff
@@ -0,0 +1,7 @@
+{
+  "MercadoPago": {
+    "AccessToken": "REPLACE_FOR_ACCESS_TOKEN",
+    "CreateTokenService": "https://api.mercadopago.com/v1/card_tokens",
```

Comment:
@diegoamatte , the ```"CreateTokenService"``` configuration you can remove from the int and qa settings because are the same that prod environment.